src/gfnff/gfnff_ini.f90

```diff
@@ -1339,8 +1344,13 @@ integer function itabrow6(i)
             if(param%group(ja).gt.2.and.imetal(jj).eq.1)mtyp2=3  ! main group
             if(imetal(jj).eq.2)                   mtyp2=4  ! TM
             qafac=topo%qa(ii)*topo%qa(jj)*25.0d0
-            dum=exp(-15.d0*qafac)/(1.0d0+exp(-15.d0*qafac))
```

Comment:
`exp(x)/(1+exp(x))` can be updated to `1/(1+exp(-x))`. In this way, these is no need for if/else conditions.

Reply:
Thanks for the hint!